tox.ini

```diff
@@ -1,69 +1,59 @@
 [tox]
-envlist = py38, pylint, format, flake8, docs
+envlist = py38, py39, py310, pylint, format, flake8, docs
 skip_missing_interpreters = true
+isolated_build = true
 
 [testenv]
-changedir = tests
-conda_deps = 
+conda_deps =
     pyibis-ami
     chaco
     # enable
 conda_channels =
     dbanas
 conda_install_args =
     --use-local
-deps = 
+deps =
+    pytest
+    pytest-xdist
     pytest-cov
-    numpy
-    Cython
-commands = 
+commands =
     py.test --basetemp={envtmpdir} -vv --cov=pybert \
-            --cov-report=html --cov-report=term-missing
-
-[testenv:pylint]
-deps = 
-    pylint
-    numpy
-    Cython
-commands = 
-    pylint --rcfile={toxinidir}/tox.ini pybert
-
-[testenv:flake8]
-deps = 
-    flake8
-    numpy
-    Cython
-commands = 
-    flake8 pybert
```

Comment:
Are these just being updated to: `lint` and `type-check`, respectively?

Reply:
Correct! When I actually started to use them, I found a few more issues that needed altering.  The diff is terrible and I apologize.  Let me see if github can clear that up.